docs/intro/tutorial01.txt

```diff
@@ -118,6 +118,182 @@ These files are:
 
 .. _more about packages: https://docs.python.org/tutorial/modules.html#packages
 
+Creating the Polls app
+----------------------
```

Comment:
Yes, at some point I got confused about it. The most part I just leave as is.
Thanks for pointing out, I will fix it later.